src/modules/uxrce_dds_client/uxrce_dds_client.cpp

```diff
@@ -422,8 +422,8 @@ UxrceddsClient::~UxrceddsClient()
 #endif // UXRCE_DDS_CLIENT_UDP
 }
 
-static void fillMessageFormatResponse(const message_format_request_s &message_format_request,
-				      message_format_response_s &message_format_response)
+static void fill_message_format_response(const message_format_request_s &message_format_request,
```

Comment:
naming convention nitpick 

https://docs.px4.io/main/en/contribute/code.html#function-and-method-names